tests/Bridge/Symfony/Routing/ApiLoaderTest.php

```diff
@@ -113,6 +134,24 @@ private function getApiLoaderWithResourceMetadata(ResourceMetadata $resourceMeta
 
         $kernelProphecy = $this->prophesize(KernelInterface::class);
         $kernelProphecy->locateResource(Argument::any())->willReturn($routingConfig);
+        $possibleArguments = [
+            'api_platform.action.get_collection',
+            'api_platform.action.post_collection',
+            'api_platform.action.get_item',
+            'api_platform.action.put_item',
+            'api_platform.action.delete_item',
+        ];
+
+        $containerInterfaceProphecy = $this->prophesize(ContainerInterface::class);
+        $containerInterfaceProphecy->reveal();
```

Comment:
I prefer PHPSpec too, and I use it for my private projects. But I'm tired of "I don't know PHPSpec so I let you write tests. Or I didn't know PHPSpec so I wrote PHPUnit tests." comments ;)